### PR TITLE
Merge pull request #808 from siddharthvp/tboptout

### PR DESCRIPTION
**Pull request by siddharthvp**
Alternative to #807.

### CODE REVIEW

File: modules/friendlytalkback.js [hunk 1]
```diff
@@ -95,28 +95,17 @@ Twinkle.talkback.callback = function() {
...
-			Twinkle.talkback.optout += ': ' + decodeURIComponent(url.substring(url.indexOf('reason=') + 7));
-		} else {
-			Twinkle.talkback.optout += '.';
-		}
-	} else {
-		Twinkle.talkback.optout = false;
-	}
-
-	var $status = $('#twinkle-talkback-optout-message');
-	if ($status.length && Twinkle.talkback.optout) {
```
Comment (siddharthvp) [inline modules/friendlytalkback.js]: both checks are unnecessary now: (i) trying to do `.text()` on empty jquery selection doesn't give any error. (ii) If not opted out, `Twinkle.talkback.optout` is just the empty string - no harm in appending it.

File: modules/friendlytalkback.js [hunk 2]
```diff
@@ -95,28 +95,17 @@ Twinkle.talkback.callback = function() {
...
-		$status.text(Twinkle.talkback.optout);
+		var reason = mw.util.getParamValue('reason', url);
```
Comment (Amorymeltzer) [inline modules/friendlytalkback.js]: Smart use of `mw.util.getParamValue`

**Comment by siddharthvp** [comment]
That was quick!

> Amorymeltzer changed the title ~~talkback: simplify talkback optout code~~ Merge pull request #808 from siddharthvp/tboptout

This shouldn't be happening, right? Been using some broken script? Same with #794 and #791.

**Comment by Amorymeltzer** [comment]
Thanks for catching that; I actually think there was something up with the GH interface, I recall seeing the "will rename when merging" when using the web interface on and off over the past week or so.  Possibly one of my chrome addons, I've got a few GH-specific ones.

**Pull request merged**